ecs-cli/modules/utils/compose/convert_task_definition.go

```diff
@@ -99,24 +124,20 @@ func ConvertToTaskDefinition(taskDefinitionName string, context *project.Context
 		containerDef := &ecs.ContainerDefinition{
 			Name: aws.String(name),
 		}
-		if err := convertToContainerDef(context, serviceConfig, volumes, containerDef); err != nil {
-			return nil, err
+
+		// Check if essential is specified for the container
+		ecsContainerDef := &ContainerDef{}
```

Comment:
See my other comment, doing this kind of makes False be the default for the essential- and I suspect that the default should be True. Let's discuss this in person.

Reply:
The default should not be true because that would sort of defeat the purpose of the feature. I suppose it comes down to whether we are assuming all containers are not essential unless otherwise specified, or that they are essential unless otherwise specified. I was going with the former, but on closer reading of #267 it looks like customers tend to want to mark one-off containers as non-essential. I'll change the code accordingly.